src/Common/Bridge/BridgeMethodsRule.php

```diff
@@ -273,4 +283,22 @@ protected function getBridgedInterfaceReflection(array $classMethods): ?Reflecti
 
         return null;
     }
+
+    /**
+     * @param array<\ReflectionParameter> $params
+     *
+     * @return int
+     */
+    protected function countRequireParams(array $params): int
+    {
+        $countParams = 0;
+
+        foreach ($params as $param) {
+            if (!$param->isDefaultValueAvailable()) {
```

Comment:
I dont think is is correct as per ticket.

Reply:
We can keep this, though. 
It seems fine to check for optional args and ignore those.